.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,19 @@
-
 ## Checklist
-- [ ] Followed the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) when writing a release note.
-- [ ] Add additional sections for `feat` and `fix` pull requests.
-- [ ] [Library documentation](https://github.com/DataDog/dd-trace-py/tree/1.x/docs) and/or [Datadog's documentation site](https://github.com/DataDog/documentation/) is updated. Link to doc PR in description.
-
-<!-- Copy and paste the relevant snippet based on the type of pull request -->
-
-<!-- START feat -->
-
-## Motivation
-<!-- Expand on why the change is required, include relevant context for reviewers -->
-
-## Design 
-<!-- Include benefits from the change as well as possible drawbacks and trade-offs -->
 
-## Testing strategy
-<!-- Describe the automated tests and/or the steps for manual testing.
+- [ ] Change(s) are motivated and described in the PR description.
+- [ ] Testing strategy is described if automated tests are not included in the PR.
+- [ ] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
+- [ ] Change is maintainable (easy to change, telemetry, documentation).
+- [ ] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) are followed.
+- [ ] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
 
-<!-- END feat -->
-
-<!-- START fix -->
-
-## Relevant issue(s)
-<!-- Link the pull request to any issues related to the fix. Use keywords for links to automate closing the issues once the pull request is merged. -->
-
-## Testing strategy
-<!-- Describe any added regression tests and/or the manual testing performed. -->
-
-<!-- END fix -->
 
 ## Reviewer Checklist
+
 - [ ] Title is accurate.
+- [ ] No unnecessary changes are introduced.
 - [ ] Description motivates each change.
-- [ ] No unnecessary changes were introduced in this PR.
-- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
-- [ ] Release note has been added and follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines), or else `changelog/no-changelog` label added.
-- [ ] All relevant GitHub issues are correctly linked.
-- [ ] Change contains telemetry where appropriate (logs, metrics, etc.).
-- [ ] Telemetry is meaningful, actionable and does not have the potential to leak sensitive data.
+- [ ] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
+- [ ] Testing strategy adequately addresses listed risk(s).
+- [ ] Change is maintainable (easy to change, telemetry, documentation).
```

Comment:
```suggestion
- [ ] Change is maintainable (easy to change, telemetry, documentation, debug).
```

?

non-blocking, we don't need to be crazy exhaustive, just is this a worth while addition to call out?

Reply:
I think this is captured a bit more specifically by `telemetry` so gonna defer on the suggestion